tests/playwright/playwright.config.js

```diff
@@ -2,7 +2,7 @@ import { resolve } from 'path';
 
 function getGrepInvert() {
 	if ( '@default' === process.env.TEST_SUITE ) {
-		return [ /@reverse-columns/, /@nested-tabs/, /@container/, /@nested-accordion/ ];
+		return [ /@reverse-columns/, /@nested-tabs/, /@container/, /@nested-accordion/, /styleguide_image_link/, /elements-regression/ ];
```

Comment:
```suggestion
		return [ /@reverse-columns/, /@nested-tabs/, /@container/, /@nested-accordion/, /@styleguide_image_link/, /@elements-regression/ ];
```

Reply:
fixed. But this is a regex, ``` @``` will not make a big difference